.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Check whether minirust succeeds to build.
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: download minirust-tooling
+        run: |
+          git clone "https://github.com/memoryleak47/minirust-tooling" ../minirust-tooling
```

Comment:
So `cargo install specr --git https://github.com/memoryleak47/minirust-tooling#19f3392` or something like that does not work? I guess the problem is that it needs to know where to find libspecr?

Reply:
As of now `minirust-tooling` requires a particular directory structure to function.
`minirust-tooling/minirust` needs to be present, it can then transpile this to `minirust-tooling/gen-minirust`, which is consumed by eg. `minimize`.

I want this to become more flexible later on, I think.
But for now it's probably easiest to `clone` it, and then soft-link `minirust` to the correct spot.

Reply:
Okay, fair.